src/modules/chat/index.js

```diff
@@ -53,6 +53,13 @@ function hasNonASCII(message) {
 
 class ChatModule {
     constructor() {
+        settings.add({
+            id: 'noChatHighlight',
+            name: 'Chat Message Highlight',
```

Comment:
this naming kind-of conflicts with bttv's builtin message highlighting .. we may want to convey this clearer as Channel Points.. maybe `Disable Channel Points Highlighted Messages` or something